binaries/.classpath_win32

```diff
@@ -4,11 +4,7 @@
 	<classpathentry kind="src" path="Eclipse SWT/win32"/>
 	<classpathentry kind="src" path="Eclipse SWT/common"/>
 	<classpathentry kind="src" path="Eclipse SWT PI/common"/>
-	<classpathentry kind="src" path="Eclipse SWT PI/win32">
-		<attributes>
-			<attribute value="org.eclipse.swt.win32.win32.x86_64" name="org.eclipse.jdt.launching.CLASSPATH_ATTR_LIBRARY_PATH_ENTRY"/>
-		</attributes>
```

Comment:
@HannesWell I just found that because of removing this attribute it is not possible to start a standalone SWT snippet from the IDE anymore. With this attribute, the `java.library.path` refers to the fragment containing the SWT dlls, which is missing without it. Was there a specific reason why you removed the attribute? Or can we reinsert it or provide some other way to run SWT standalone snippets easily?

Reply:
Same on Mac.

Reply:
@HannesWell @HeikoKlare  If the attribute is re-instated then on Mac there needs to be two `.classpath_cocoa` files - one for `aarch64` and one for `x86_64`.

Reply:
Yes, the change would need to be done in every fragment's classpath file.

Reply:
Indeed. When implementing this change I only tested it with an Eclipse application and for an OSGi runtime this was not necessary, so I assumed it was a left-over from old times.
But testing it now with a standalone SWT snippet I also see it fails, sorry for the disturbance.

But at the same time this was a welcome simplification since it allowed to re-use the same .classpath file for all macos platforms. At the `.classpath` file for Linux/GTK only referenced the x86_64 arch project so it will probably not have worked for development on all platforms.
To make it work we would probably have provide individual .classpath files for each fragment.
I have opened https://github.com/eclipse-platform/eclipse.platform.swt/pull/1031 to investigate if we can use a variable for that, but it currently seems not to work.